webpack.config.js

```diff
@@ -1,34 +1,45 @@
 const path = require('path')
+const webpack = require('webpack')
 
 module.exports = {
-  mode: 'production',
   entry: {
     app: './src/app.ts',
-    skydb: './src/skydb.ts',
-  },
-  output: {
-    filename: '[name].js',
-    path: path.resolve(__dirname, './dist'),
-    libraryTarget: 'umd',
-    umdNamedDefine: true,
-  },
-  resolve: {
-    extensions: ['.ts', '.js'],
   },
   module: {
     rules: [
       {
         test: /\.tsx?$/,
-        use: 'ts-loader',
+        loader: 'esbuild-loader',
         exclude: /node_modules/,
+        options: {
+          loader: 'tsx',
+          target: 'es2020',
+        },
       },
     ],
   },
-  node: {
-    console: false,
-    fs: 'empty',
-    net: 'empty',
-    tls: 'empty',
-    child_process: 'empty',
+  resolve: {
+    extensions: ['.ts', '.js'],
+    fallback: {
+      stream: require.resolve('stream-browserify'),
```

Comment:
I think we need `stream` to run Ceramic, but all the others could be set to `false`, no?

Reply:
I thought so too, unfortunately this all seems to be required. It might be worth trying to update some of the packages that we install but this entire thing was set to false & I went through and one by one enabled to make sure that I didn't have anything that wasn't needed there.

Reply:
OK thanks, might be dependencies from the wallets, no? Maybe by updating them these dependencies on fallback modules can be reduced?